packages/zwave-js/src/lib/commandclass/EntryControlCC.ts

```diff
@@ -0,0 +1,683 @@
+key cached size max:    ${resp.keyCachedSizeMax}
+key cached timeout min: ${resp.keyCachedTimeoutMin}
+key cached timeout max: ${resp.keyCachedTimeoutMax}`,
+					direction: "inbound",
+				});
+			}
+		}
+
+		this.driver.controllerLog.logNode(node.id, {
+			endpoint: this.endpointIndex,
+			message: "requesting entry control configuration...",
+			direction: "outbound",
+		});
+
+		{
+			const resp = await api.getConfiguration();
+			if (resp) {
+				this.driver.controllerLog.logNode(node.id, {
+					endpoint: this.endpointIndex,
+					message: `received entry control supported keys:
+key cached size:    ${resp.keyCachedSize}
+key cached timeout: ${resp.keyCachedTimeout}`,
+					direction: "inbound",
+				});
+			}
+		}
+
+		// Remember that the interview is complete
+		this.interviewComplete = true;
+	}
+}
+
+@CCCommand(EntryControlCommand.Notification)
+export class EntryControlCCNotication extends EntryControlCC {
+	public constructor(
+		driver: Driver,
+		options: CommandClassDeserializationOptions,
+	) {
+		super(driver, options);
+
+		validatePayload(this.payload.length >= 4);
+		this._sequenceNumber = this.payload[0];
+		this._dataType = this.payload[1] & 0b11;
+		this._eventType = this.payload[2];
+		const eventDataLength = this.payload[3];
+
+		validatePayload(this.payload.length >= 4 + eventDataLength);
+		this._eventData = Buffer.from(
+			this.payload.slice(4, 4 + eventDataLength),
+		);
+	}
+
+	private _sequenceNumber: number;
+	public get sequenceNumber(): number {
+		return this._sequenceNumber;
+	}
+
+	private _dataType: EntryControlDataTypes;
+	public get dataType(): EntryControlDataTypes {
+		return this._dataType;
+	}
+
+	private _eventType: EntryControlEventTypes;
+	public get eventType(): EntryControlEventTypes {
+		return this._eventType;
+	}
+
+	private _eventData: Buffer;
+	public get eventData(): Buffer {
+		return this._eventData;
+	}
+
+	public toLogEntry(): MessageOrCCLogEntry {
+		const message: MessageRecord = {
+			"sequence number": this.sequenceNumber,
+			"data type": this.dataType,
+			"event type": this.eventType,
+			"event data": this.eventData.toString(),
+		};
+		return {
+			...super.toLogEntry(),
+			message,
+		};
+	}
+}
+
+@CCCommand(EntryControlCommand.KeySupportedReport)
+export class EntryControlCCKeySupportedReport extends EntryControlCC {
+	public constructor(
+		driver: Driver,
+		options: CommandClassDeserializationOptions,
+	) {
+		super(driver, options);
+
+		validatePayload(this.payload.length >= 1);
+		const length = this.payload[0];
+		validatePayload(this.payload.length >= 1 + length);
+		this._keySupported = parseBitMask(this.payload.slice(1, 1 + length), 0);
+		this.persistValues();
+	}
+
+	private _keySupported: number[];
+
+	@ccValue()
+	@ccValueMetadata({
+		...ValueMetadata.ReadOnly,
+		type: "number[]",
+		label: "Supported keys",
+		description: "A list of supported keys",
+	})
+	public get keySupported(): number[] {
+		return this._keySupported;
+	}
```

Comment:
```suggestion
    @ccValue({internal: true})
	public readonly supportedKeys: readonly number[];
```
The support information is internal to zwave-js. UIs don't want to display this without context.